kitsune/sumo/tests/__init__.py

```diff
@@ -126,6 +126,15 @@ def setUp(self):
         self.client.cookies[settings.MOBILE_COOKIE] = 'on'
 
 
+class MinimalViewTestCase(TestCase):
+    def setUp(self):
+        super(MinimalViewTestCase, self).setUp()
+
+    def get_minimal_url(self, document):
+        minimalurl = '?minimal=1&mobile=1'
+        return document.get_absolute_url()+minimalurl
```

Comment:
Add spaces between the operator and operands please (IE `document.get_absolute_url() + minimalurl`).

@rehandalal I don't know enough about triggering the minimal view to judge whether this is a fine way of doing so. Thoughts?